curations/maven/mavengoogle/com.google.android.gms/play-services-auth-base.yaml

```diff
@@ -13,3 +13,6 @@ revisions:
   18.0.4:
     licensed:
       declared: OTHER
+  18.0.5:
+    licensed:
+      declared: NONE
```

Comment:
Looks like this should be updated to OTHER instead of NONE like the other license details for this library.